### PR TITLE
French update of the MWC 2017 wiki page

### PR DESCRIPTION
**Pull request by Mathisca**
### Description

Update the translation of the MWC 2017 French wiki page. I followed the same commits as in the English version.

### Status

Finished

### Changelog

- Add QF results
- Add SF mappool/schedule
- Fix arrays and add clarification about loser bracket

### CODE REVIEW

File: wiki/Tournaments/MWC/2017/4K/fr.md [hunk 1]
```diff
@@ -48,25 +48,37 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
 
-### Samedi 26 août 2017
+### Samedi 2 septembre 2017
 
 | Équipe A |  |  | Équipe B | Heure du match (UTC) | Heure locale A | Heure locale B |
-| ---: | ---: | :--- | :--- | :---: | :---: | :---: |
-| France | ![][flag_FR] | ![][flag_CL] | Chili | **19h00 UTC** | 21h00 UTC +2 | 16h00 UTC -3 |
-| Argentine | ![][flag_AR] | ![][flag_GB] | Royaume-Uni | **20h30 UTC** | 17h30 UTC -3 | 21h30 UTC +1 |
-| Canada | ![][flag_CA] | ![][flag_US] | États-Unis | **22h00 UTC** | 17h00 UTC -5 | 17h00 UTC -5 |
+| ---: | ---: | :--- | :--- | :---: | :---: | :---: | |
```
Comment (SpartanPlume) [inline wiki/Tournaments/MWC/2017/4K/fr.md]: Copie colle ça : `| ---: | ---: | :--- | :--- | :---: | :---: | :---: | - |`

File: wiki/Tournaments/MWC/2017/4K/fr.md [hunk 2]
```diff
@@ -48,25 +48,37 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+### Dimanche 3 septembre 2017
 
 | Équipe A |  |  | Équipe B | Heure du match (UTC) | Heure locale A | Heure locale B |
-| ---: | ---: | :--- | :--- | :---: | :---: | :---: |
-| Corée du Sud | ![][flag_KR] | ![][flag_JP] | Japon | **9h00 UTC** | 18h00 UTC +9 | 18h00 UTC +9 |
-| Nouvelle-Zélande | ![][flag_NZ] | ![][flag_PL] | Pologne | **10h30 UTC** | 22h30 UTC +12 | 12h30 UTC +2 |
-| Singapour | ![][flag_SG] | ![][flag_CN] | Chine | **12h00 UTC** | 20h00 UTC +8 | 20h00 UTC +8 |
-| Hong-Kong | ![][flag_HK] | ![][flag_SE] | Suède | **13h30 UTC** | 21h30 UTC +8 | 15h30 UTC +2 |
-| Brésil | ![][flag_BR] | ![][flag_ID] | Indonésie | **15h00 UTC** | 12h00 UTC -3 | 22h00 UTC +7 |
+| ---: | ---: | :--- | :--- | :---: | :---: | :---: | |
```
Comment (SpartanPlume) [inline wiki/Tournaments/MWC/2017/4K/fr.md]: Copie colle ça : `| ---: | ---: | :--- | :--- | :---: | :---: | :---: | - |`

File: wiki/Tournaments/MWC/2017/4K/fr.md [hunk 3]
```diff
@@ -48,25 +48,37 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+| Japon | ![][flag_JP] | ![][flag_ID] | Indonésie | **12h30 UTC** | 21h30 UTC +9 | 19h30 UTC +7 | ³ |
+| Chine | ![][flag_CN] | ![][flag_GB] | Royaume-Uni | **14h00 UTC** | 22h00 UTC +8 | 15h00 UTC +1 | ³ |
+| Chine | ![][flag_CN] | ![][flag_CA] | Canada | **14h00 UTC** | 22h00 UTC +8 | 9h00 UTC -5 | ³ |
+| France | ![][flag_FR] | ![][flag_GB] | Royaume-Uni | **14h00 UTC** | 16h00 UTC +2 | 15h00 UTC +1 | ³ |
+
+¹ Rencontre du tableau des gagnants
+
+² Rencontre du tableau des perdants
+
+³ La rencontre dépend du résultat de la ²rencontre du tableau des perdants
```
Comment (SpartanPlume) [inline wiki/Tournaments/MWC/2017/4K/fr.md]: ²rencontre -> ²Rencontre
Pour bien comprendre qu'on parle du ²

Comment (Mathisca) [reply wiki/Tournaments/MWC/2017/4K/fr.md]: merci pour la review ! :+1:

**Pull request merged**